### DISCUSSION TITLE
`.flake8_nb` does not work in certain cases

### DISCUSSION THREAD

**Discussion by jbloom** [s-weigand/flake8-nb#243] [Q&A] [answered]
I'm using `flake8_nb` version 0.4.0.

I have a subdirectory in my project called `./docs/` which contains Jupyter notebooks that fail the flaking, e.g. `./docs/fail_flake8.ipynb`.

My `.flake8_nb` configuration file includes `docs/*`:
```
[flake8_nb]
exclude =
    .*,
    _*,
    docs/*
```

Nonetheless, `flake8_nb` reports failures on the `./docs/fail_flake8.ipynb` notebook.

**Comment by s-weigand** [comment, marked as answer]
If you use `docs` instead of `docs/*` it should work.
I tried `docs/**/*` and it also doesn't work, looks like this is an upstream issue with `flake8` (at least on windows where it expands the pattern to a path with `\\`).
Also, you should rather use [`extend-exclude`](https://flake8.pycqa.org/en/latest/user/options.html?highlight=extend%20ignore#cmdoption-flake8-extend-exclude) instead of `exclude`, that way you won't overwrite the default values of `exclude` (`['.svn', 'CVS', '.bzr', '.hg', '.git', '__pycache__', '.tox', '.eggs', '*.egg', '.ipynb_checkpoints']`)

If the notebook is failing because you want to show off some bad practice, you could just ignore those cells/the specific error in those cells (see [cell-tags](https://flake8-nb.readthedocs.io/en/latest/usage.html#cell-tags)), that way you can show off the bad practice without accidentally adding some yourself in other places of the notebooks.

Feel free to close this issue if this resolves your problem.

**Comment by jbloom** [comment]
That worked, thanks!
